src/test/java/com/scottlogic/SortingTests/AuthorSortingTests.java

```diff
@@ -0,0 +1,72 @@
+package com.scottlogic.SortingTests;
```

Comment:
A suggestion for test packages: keep it the same as the classes you test.

E.g.
main -> java -> com -> scottlogic -> AuthorSorting
test -> java -> com -> scottlogic -> AuthorSortingTests

It's a convention so people can find the test classes easier and makes imports a bit neater.
You already declared at the top level that it is a test folder, no need to add a 'Test' suffix in your packages too.